src/.nuget/dir.props

```diff
@@ -149,6 +162,14 @@
       <Platform>arm64</Platform>
     </OfficialBuildRID>
   </ItemGroup>
+  <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Android;'))">
+    <OfficialBuildRID Include="android.21-arm64">
+      <Platform>arm64</Platform>
+    </OfficialBuildRID>
+    <OfficialBuildRID Include="android.21-arm">
```

Comment:
I've only tested...and am only able to test...for arm64.  I can add arm into this change but won't be able to test it.